test/test_jsonpath.rb

```diff
@@ -48,13 +48,17 @@ def test_retrieve_all_prices
   end
 
   def test_recognize_array_splices
-    assert_equal [@object['store']['book'][0], @object['store']['book'][1]], JsonPath.new('$..book[0:1:1]').on(@object)
+    assert_equal [@object['store']['book'][0]], JsonPath.new('$..book[0:1:1]').on(@object)
```

Comment:
@Skarlso should I add or replace this line here `assert_equal [@object['store']['book'][0], @object['store']['book'][1]], JsonPath.new('$..book[0:2:1]').on(@object)`?

Reply:
Add one. :)

Reply:
Done!